File: app/src/main/java/com/esoxjem/movieguide/listing/sorting/SortingDialogView.java
package com.esoxjem.movieguide.listing.sorting;

/**
 * @author arun
 */
interface SortingDialogView
{
    void setPopularChecked();

    void setNewestChecked();

    void setHighestRatedChecked();

    void setFavoritesChecked();

    void dismissDialog();

}

File: app/src/main/java/com/esoxjem/movieguide/movies/sorting/SortingDialogView.kt
package com.esoxjem.movieguide.movies.sorting

/**
 * @author arunsasidharan
 */
interface SortingDialogView {
    fun setPopularChecked()

    fun setNewestChecked()

    fun setHighestRatedChecked()

    fun setFavoritesChecked()

    fun dismissDialog()

}
